### PR TITLE
Reference v1.0.0.html and create a copy of v1 images

### PR DESCRIPTION
**Pull request by felicialim**
***
<a href="https://pr-preview.s3.amazonaws.com/felicialim/iamf/pull/777.html" title="Last updated on Dec 1, 2023, 2:44 AM UTC (48e1844)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/777/cb5b4b2...felicialim:48e1844.html" title="Last updated on Dec 1, 2023, 2:44 AM UTC (48e1844)">Diff</a>

### CODE REVIEW

**Review by sunghee-hwang — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -7,6 +7,7 @@ Editor: Felicia Lim, Google, flim@google.com
 Repository: AOMediaCodec/iamf
 Shortname: iamf
 URL: https://aomediacodec.github.io/iamf/
+Previous Version: https://aomediacodec.github.io/iamf/v1.0.0.html
```
Comment (sunghee-hwang) [inline index.bs:10]:
```suggestion
Previously approved version: https://aomediacodec.github.io/iamf/v1.0.0.html
Latest approved version: https://aomediacodec.github.io/iamf/latest-approved.html
Latest draft version: https://aomediacodec.github.io/iamf/latest-draft.html
```

File: v1.0.0.html [hunk 2]
```diff
@@ -1947,7 +1947,9 @@ <h3 class="no-num no-toc no-ref heading settled" id="profile-and-date"><span cla
    <div data-fill-with="spec-metadata">
     <dl>
      <dt>This version:
-     <dd><a class="u-url" href="https://aomediacodec.github.io/iamf/">https://aomediacodec.github.io/iamf/</a>
+     <dd><a class="u-url" href="https://aomediacodec.github.io/iamf/v1.0.0.html">https://aomediacodec.github.io/iamf/v1.0.0.html</a>
+     <dt>Latest draft version:
+     <dd><a class="u-url" href="https://aomediacodec.github.io/iamf/index.html">https://aomediacodec.github.io/iamf/index.html</a>
```
Comment (sunghee-hwang) [inline v1.0.0.html:1952]:
```suggestion
     <dt>Latest approved version:
     <dd><a class="u-url" href="https://aomediacodec.github.io/iamf/index.html">https://aomediacodec.github.io/iamf/latest-approved.html</a>
     <dt>Latest draft version:
     <dd><a class="u-url" href="https://aomediacodec.github.io/iamf/index.html">https://aomediacodec.github.io/iamf/latest-draft.html</a>
```

**Review by sunghee-hwang — COMMENTED** [review]
To implement the proposed header and keep the consistency between HTML files.

File: v1.0.0.html (hunk 2 above)
Comment (felicialim) [reply v1.0.0.html]: thanks, I edited directly to use your suggestion, but also update the href links.

File: index.bs [hunk 3]
```diff
@@ -7,6 +7,9 @@ Editor: Felicia Lim, Google, flim@google.com
 Repository: AOMediaCodec/iamf
 Shortname: iamf
 URL: https://aomediacodec.github.io/iamf/
+!Previously approved version: https://aomediacodec.github.io/iamf/v1.0.0.html
+!Latest approved version: https://aomediacodec.github.io/iamf/latest-approved.html
+!Latest draft version: https://aomediacodec.github.io/iamf/latest-draft.html
```
Comment (sunghee-hwang) [inline index.bs:12]:
```suggestion
!Previously approved version: <a href="https://aomediacodec.github.io/iamf/v1.0.0.html">https://aomediacodec.github.io/iamf/v1.0.0.html</a>
!Latest approved version: <a href="https://aomediacodec.github.io/iamf/latest-approved.html">https://aomediacodec.github.io/iamf/latest-approved.html</a>
!Latest draft version: <a href="https://aomediacodec.github.io/iamf/latest-draft.html">https://aomediacodec.github.io/iamf/latest-draft.html</a>
```

**Pull request merged**